otx/algorithms/classification/tasks/inference.py

```diff
@@ -347,22 +363,23 @@ def patch_color_conversion(pipeline):
                     patch_color_conversion(pipeline_step.transforms)
 
         assert "data" in config
-        for subset in ("train", "val", "test"):
+        for subset in ("train", "val", "test", "unlabeled"):
             cfg = config.data.get(subset, None)
             if not cfg:
                 continue
             if cfg.type == "RepeatDataset":
```

Comment:
Is this `RepeatDataset` still used in classification? If it's not being used, it's good to remove it.

Reply:
It's out of scope of this PR. I hope it can be dealt in other.

Reply:
We just need to delete this line  This is something we missed in our previous cls task refactoring.

Reply:
But still I don't think it needs to deal in semi supervised learning related PR. It still remains in develop branch. I can make comments TODO sign for it.